src/Umbraco.Web/Runtime/WebRuntimeComponent.cs

```diff
@@ -278,10 +282,7 @@ private static void ConfigureClientDependency(IGlobalSettings globalSettings)
                 { "compositeFileHandlerPath", ClientDependencySettings.Instance.CompositeFileHandlerPath }
             });
 
-            // When using a non-web runtime and this component is loaded ClientDependency explodes because it'll
-            // want to access HttpContext.Current, which doesn't exist
-            if (IOHelper.IsHosted)
```

Comment:
@aaronpowell you don't want to keep this check?

Reply:
I moved it up to line 78, which is the call to the function that that was in, so no value in doing it twice.

Reply:
Yup but just want to make sure we don't want to check twice? The first one prevents the call to ` ConfigureClientDependency(_globalSettings);` and the 2nd one prevents the call to `ClientDependencySettings.Instance.MvcRendererCollection.Add(renderer);` or does the MvcRenderCollection.Add not care if it has an http context?

Reply:
I'd actually need to lift it above line 282 where the `compositeFileHandlerPath` is, since that also uses `ClientDependencySettings.Instance`. So that's why I threw it outside the method and didn't bother to do an internal check.

Reply:
ah of course, i didn't actually look at the entire file with the changeset :P  all good, will merge

Reply:
🤣